### PR TITLE
Update cross-tenant-mailbox-migration.md

### PR DESCRIPTION
**Pull request by zouma83**
### How do I know this worked?
=> testing relationship^and endpoint before/after creating at list one object in the target tenant
;)
FM

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by JohanFreelancer9 — COMMENTED** [review]
@yogkumgit - Could you please request a review from the document author or manager before I do a copy review. I am not at all sure about this.

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please help. Thanks

**Comment by zouma83** [comment]
hi all

tested and used twice during two migrations with the actual manual
> Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]" -TestMailbox "[email address of a source mailbox that is part of your migration scope]"

nok error, even before/after creating target object(s)
cf. command's documentation Test-MigrationServerAvailability seems not working with -Endpoint and -TestMailbox parameters together


so for checking the source & target organisation's preparations:
target tenant:
> Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]"
> Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability

source tenant:
> Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability

ok, informations indicate ready for mailboxes migrations for both organisations (source & target)
these commands came from the previous manual's version (tested and used several times too)

thanks

File: microsoft-365/enterprise/cross-tenant-mailbox-migration.md [hunk 1]
```diff
@@ -206,7 +206,17 @@ To obtain the tenant ID of a subscription, sign in to the [Microsoft 365 admin c
...
+   > - target tenant:
+   > 
+   > Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]"
+   >
+   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability
+   >
+   > - source tenant:
+   > 
+   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability
+
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/cross-tenant-mailbox-migration.md]:
```suggestion
   > - Target tenant:
   > 
   > Test-MigrationServerAvailability -Endpoint "[the name of your cross-tenant migration endpoint]"
   >
   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability
   >
   > - Source tenant:
   > 
   > Get-OrganizationRelationship | fl name, DomainNames, MailboxMoveEnabled, MailboxMoveCapability 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @zouma83, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**